### DISCUSSION TITLE
求助：为什么我用透视表，度量值有多个的时候，但最后只展示最后一行数据呢？而非多个数据进行求和

### DISCUSSION THREAD

**Discussion by Baili-BL** [antvis/S2#2526] [General]
![image](https://github.com/antvis/S2/assets/35422348/362e0ca3-63b1-429e-9f54-e70c3088d429)

求助：为什么我用透视表，度量值有多个的时候，但最后只展示最后一行数据呢？而非多个数据进行求和，希望大佬帮忙看下，感谢。

**Comment by wjgogogo** [comment]
给一个复现链接，描述信息太少了
